generators/client/templates/angular/src/main/webapp/app/admin/user-management/user-management.module.ts.ejs

```diff
@@ -0,0 +1,45 @@
+<%#
+ Copyright 2013-2019 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+
+import { <%= angularXAppName %>SharedModule } from 'app/shared/shared.module';
+import { UserMgmtComponent } from './user-management.component';
+import { UserMgmtDetailComponent } from './user-management-detail.component';
+import { UserMgmtUpdateComponent } from './user-management-update.component';
+import { UserMgmtDeleteDialogComponent } from './user-management-delete-dialog.component';
+
+@NgModule({
```

Comment:
As we are creating a separate module, we should also encapsulate user management routes inside that (lazy/eager - either should be fine).

Reply:
My main purpose is to create load UserManagementComponent into another menu/module with a different role.
I will take a look at what can be done.

By the way, UserResolve doesn't seems to be used anywhere, am I right?
https://github.com/jhipster/generator-jhipster/blob/7e5e5f770efa686886692c5a2a842f6c4934acd4/generators/client/templates/angular/src/main/webapp/app/admin/user-management/user-management.route.ts.ejs#L30-L38

Reply:
Yes, `UserResolve` is redundant. I suggested to remove this in: https://github.com/jhipster/generator-jhipster/pull/10383#issuecomment-530793229  
But I believe that this can be removed as part of this PR.